### DISCUSSION TITLE
视频的评论显示不完整

### DISCUSSION THREAD

**Discussion by qinghe9457** [ywmoyue/biliuwp-lite#642] [feature]
### 问题描述

希望视频评论能完整的呈现

### 描述解决方案

对过长字数评论增加查看按钮

### 附加信息

无

### 代码贡献

否

**Comment by GD-Slime** [comment]
你好，能否发个图呢？ 哪里有评论不完整的地方？ 
一般来说，过长的评论会有展开按钮，展开后即可看到完整评论。
